tests/modules/nf-core/bedtools/intersect/main.nf

```diff
@@ -13,7 +13,7 @@ workflow test_bedtools_intersect {
 
     extension = 'bed'
 
-    BEDTOOLS_INTERSECT ( input, extension )
+    BEDTOOLS_INTERSECT ( input, extension, [] )
```

Comment:
```suggestion
    BEDTOOLS_INTERSECT ( input, extension, [[:],[]] )
```

Reply:
This means that all users that will use bedtools intersect in its usual way, without -g option will have to add empty tuples instead of empty list. 
I think its a bit tedious...

Reply:
Kind of the same thing no? Adding `[]` or `[[:],[]]` is just a couple of tokens more